setup.py

```diff
@@ -164,16 +248,42 @@ def runtest(self):
     license="BSD",
     long_description="",
     name="cellprofiler",
+    options = {
+        "py2exe": {
```

Comment:
We should move py2exe options into a `[py2exe]` block in our setup.cfg and keep setup.py multi-platform.